about.html

```diff
@@ -0,0 +1,157 @@
+              <li class="nav-list_item ggp-restaurant"><a href="#">GGP Food&Drink</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="bottom-nav_hamburger" id="hambuger">
+          <img src="./images/hamburger.svg" alt="hamburger" class="open ham-icon" id="openHamburger"/>
+          <img src="./images/closeicon.svg" alt="hambuger" class="close hidden ham-icon" id="closeHamburger"/>
+        </div>
+      </nav>
+      <div class="about_restaurant" id="about_restaurant">
+        <h3 class="about_restaurant-title">"A haven for food lovers, where every dish tells a unique story."</h3>
+        <h1 class="about_restaurant-heading">
+          Gyi Gyi Paw <br />
+          Food and Drink Restaurant
+        </h1>
+        <p class="about_restaurant-body">
+          "Welcome to Gyi Gyi Paw Restaurant, where culinary excellence meets a warm 
+          and inviting atmosphere.
+          Prepare to embark on a gastronomic journey that will delight your senses 
+          and leave a lasting impression.
+          At Gyi Gyi Paw Restaurant, our talented chefs craft a fusion of traditional 
+          and innovative dishes inspired by the rich culinary traditions of cuisine,
+          the finest local ingredients and expert craftsmanship."
+        </p>
+        <p class="about_restaurant-contact">
+          Please contact us per email for any informations and advice.
+        </p>
+        <p class="about_restaurant-email">
+          <a href="#">gyigyipaw@gmail.com</a>
+        </p>
+      </div>
+    </header>
+    <main>
+      <section class="describe">
+        <div class="describe_container">
+          <h2 class="describe_container-title">
+            Gyi Gyi Paw Food and Drink Restaurant
+          </h2>
+          <p class="describe_container-body">
+            "Step into an oasis of elegance and charm as you enter our thoughtfully
+             designed space, adorned with stylish decor and soft lighting. Whether 
+             you're celebrating a special occasion or enjoying a casual evening out, 
+             our warm and inviting atmosphere sets the perfect stage for an unforgettable 
+             dining experience."
+          </p>
+          <div class="describe_container-container">
+            <h3 class="about-logo">GGP Food&Drink</h3>
+            <div class="describe_container-container-details">
+              <h2 class="describe_container-container-details-text">Daily-Open</h2>
+              <p class="describe_container-container-details-body">
+                9:00AM ~ 1:00PM
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="previous">
+        <div class="previous_container">
+          <h4 class="previous_container-header">
+            See The Activity and Event In Restaurant
+          </h4>
+          <p class="previous_container-body">
+            Take a look at the last two event which took place in 
```

Comment:
- [ ] Please consider adding the line below the text here as you see in your case you forgot to add the line, so it will be better if you add it to make it nice like the original design 👌
- [ ] Also please try to increase the margin between the elements here to be like the Behance design as you see the elements of this section needed some margin from both top and bottom to make it nice like the original design 💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/9d2ad272-6b78-4b33-ba47-00cba29ee1d9)  |    ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/aea41b82-337c-4a61-bf0b-1dc3162a0a85)  |